marklogic-data-hub-central/ui/e2e/README.md

```diff
@@ -4,28 +4,39 @@ This hc-qa-project is intended to run HC e2e test.
 
 Run the setup script first from under `marklogic-data-hub-central/ui/e2e` directory, to initialize, deploy and run flows:
 
-    ./setup.sh dhs=<true/false> mlHost=<curatoinEndpoint/localhost>
- 
+    For DHS -
```

Comment:
@SameeraPriyathamTadikonda can you pls verify the steps noted here for DHS is what we follow in  Jenkins as well for e2e tests running against DHS.